core/src/main/java/org/apache/calcite/sql/dialect/BigQuerySqlDialect.java

```diff
@@ -833,12 +833,7 @@ private void unparseOtherFunction(SqlWriter writer, SqlCall call, int leftPrec,
       unparseCall(writer, parseTimestampCall, leftPrec, rightPrec);
       break;
     case "INSTR":
-      final SqlWriter.Frame frame = writer.startFunCall("STRPOS");
-      writer.sep(",");
-      call.operand(0).unparse(writer, leftPrec, rightPrec);
-      writer.sep(",");
-      call.operand(1).unparse(writer, leftPrec, rightPrec);
-      writer.endFunCall(frame);
+      unparseInstrFunctionCall(writer, call, leftPrec, rightPrec);
```

Comment:
unparse for "INSTR" can now be removed since we are no longer using STRPOS

Reply:
done